README

```diff
@@ -4,13 +4,15 @@ To use:
 
 * Clone ceph-ansible: https://github.com/ceph/ceph-ansible.git
 
+* install python virtualenv package (exact package name may vary) unless you are using python3
+
 * Setup a virtualenv with linode-python:
 
-    $ virtualenv2 linode-env && source linode-env/bin/activate && pip install linode-python
+    $ virtualenv linode-env && source linode-env/bin/activate && pip install linode-python
```

Comment:
I'm not a python expert but my understanding was that `virtualenv2` sets up a Python 2.X environment (installs Python 2, not 3, etc.). Is `virtualenv2` not available for you?